### DISCUSSION TITLE
Possibly support Lance as a table

### DISCUSSION THREAD

**Discussion by MrPowers** [unitycatalog/unitycatalog#444] [Ideas]
[Lance is a columnar data format](https://github.com/lancedb/lance).

UC could support it as a volume, or table, or both.  What's the best path forward here?

**Comment by SemyonSinchenko** [comment]
Why not like Vector DB? As I remember, one of the main usage of Lance is for storing embeddings as in vector db: https://docs.llamaindex.ai/en/stable/examples/vector_stores/LanceDBIndexDemo/

**Comment by everettVT** [comment]
Lance is an interesting one. You could define a volume for LanceDB to use, or you could try supporting the api natively. The great thing about lance is that it stores the raw data next to the embeddings. It's also tightly integrated with the apache arrow stack. 

I'm personally planning on using LanceDB and UC and would support a full integration following 0.2 release
